README.md

```diff
@@ -13,10 +13,28 @@ Watch demo:
 
 ## Install
 
+### Binaries
+
 Download the latest binary file at
 [Releases](https://github.com/munshkr/hydra-superdirt/releases) corresponding to
 your operating system (Linux, MacOs and Windows supported) and put it somewhere.
 
+### Install and Run Using Yarn
+
+Install [Yarn](https://yarnpkg.com )
+
+Clone repo somewhere
+
+Go to the repo folder on your terminal and run yarn
+```c
+yarn --start
```

Comment:
I think here the command is simply `yarn` or `yarn install`

Reply:
Fixed